_posts/2022-09-21-#25880.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: pr
+date: 2022-09-21
+title: "Make stalling timeout adaptive during IBD"
+pr: 25880
+authors: [mzumsande]
+components: ["p2p"]
+host: glozow
+status: upcoming
+commit: 48e5385
+---
+
+## Notes
+
+- During block download, nodes have a `BLOCK_DOWNLOAD_WINDOW=1024`-block window within their best
+  chain (this happens after downloading the full headers chain) and download those blocks in
+parallel.
+
+    - Blocks still need to be validated sequentially, otherwise some UTXOs might not be available.
```

Comment:
```suggestion
    - Blocks still need to be validated sequentially, since their validity depends on all outputs they spend being confirmed in previous blocks.
```